src/client/components/Autocomplete/Autocomplete.tsx

```diff
@@ -16,6 +17,7 @@ type Option = {
 type Props = {
   items: Array<Option>
   onSave: (value: string | any) => void
+  onPaste?: OnPaste
```

Comment:
It would be preferred not to import a type from that path.
Can it be defined inline directly ? ( in two points in this pr ) 
What do you think ?